forge.config.js

```diff
@@ -1,9 +1,11 @@
 const path = require('path');
 const fs = require('fs');
 const packageJson = require('./package.json');
+const contributors = require('./tools/contributors');
```

Comment:
nit: `const { maybeFetchContributors } = require('./tools/contributors');` and avoid the awkward `contributors` name.